src/actions/compass.ts

```diff
@@ -0,0 +1,45 @@
+import {SpecQueryModelGroup} from 'compassql/build/src/model';
+import {Query} from 'compassql/build/src/query/query';
+import {Schema} from 'compassql/build/src/schema';
+import {Dispatch} from 'redux';
+import {ThunkAction} from 'redux-thunk';
+import {ReduxAction} from './redux-action';
+
+import {fetchCompassRecommends} from '../api/api';
+import {State} from '../models/index';
+import {getQuery, getSchema} from '../selectors';
+import {Action} from './index';
+
+export type CompassAction = CompassRecommendsRequest | CompassRecommendsRecieve;
```

Comment:
I think it is better to use `Query` prefix than `Compass` as we are querying visualizations from CompassQL (In any case, it's CompassQL, not the old Compass anyway).